### PR TITLE
Profile/chore/36951/component updates

### PR DESCRIPTION
**Pull request by mdewey**
## Description
- Updated  to `va-addtional-info`
- Also fixed other linting errors in the files that were touched. 

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/36951

## Testing done
- all tests pass

### CODE REVIEW

**Review by tmitchellgcio — APPROVED** [review]

File: src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx [hunk 1]
```diff
@@ -27,44 +25,39 @@ import { transformServiceHistoryEntryIntoTableRow } from '../../helpers';
...
-    />
+    <>
+      <va-alert status="warning">
+        <h2 slot="headline">We don’t seem to have your military records</h2>
+
+        <p>
+          We’re sorry. We can’t match your information to our records. If you
+          think this is an error, please call the VA.gov help desk at{' '}
+          <va-telephone contact={CONTACTS.HELP_DESK} /> (TTY:{' '}
+          <va-telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
```
Comment (adamwhitlock1) [inline src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx:36]: Probably not important for this PR, but I don't think the `pattern` prop is supported in the web component

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-telephone/va-telephone.tsx

File: src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js [hunk 2]
```diff
@@ -173,19 +175,19 @@ describe('Direct Deposit', () => {
...
+      // saveNewBankInfo('EDU');
+      // // This scan will be run while the bank info is saving and the
+      // // LoadingButton is in its "loading" state. This would throw an aXe error
+      // // if LoadingButton.loadingText was not set
+      // cy.axeCheck();
+      // // Now wait for the update API calls to resolve to failures...
+      // cy.findAllByText(/we couldn’t update your bank info/i).should(
+      //   'have.length',
+      //   '2',
+      // );
```
Comment (adamwhitlock1) [inline src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js:190]: Should we plan on removing these completely at a later data?

**Review by adamwhitlock1 — APPROVED** [review]
Had some minor non-blocking comments, but otherwise lgtm

File: src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx (hunk 1 above)
Comment (bkjohnson) [reply src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx:36]: Adam is right - you can see the props the component supports [on its documentation page](https://design.va.gov/components/telephone#code-usage) as well as on the [storybook page](https://design.va.gov/storybook/?path=/docs/components-va-telephone--default).

Comment (adamwhitlock1) [reply src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx:36]: That is amazing @bkjohnson ! Thanks for the tip.

File: src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js (hunk 2 above)
Comment (mdewey) [reply src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js]: Ooops, That should be commented back in. Thanks for the catch!

**Pull request merged**